code/modules/mob/living/carbon/human/life/handle_disabilities.dm

```diff
@@ -73,7 +73,17 @@
 						"FUCK IT; KISSYOUR ASSES GOOD BYE DEAD MEN! I AM SELFDESTRUCKTING THE STATION!!!!", \
 						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!"))
 				if(3)
-					emote("drool")
+					// We need to find a mob in range for that ; could be useful for custom emotes that take one random mob at random as an argument, maybe ?
+					var/list/L
+					var/retardation
+					for(var/mob/M in range(1, src))
+    					L.Add(M)
+    				var/mob/mob1 = L[1]
```

Comment:
I don't think you need to cast your pick as a `/mob` in order to insert it into the string. `"pecks \the [L[1]]."` should work, I think.